### ISSUE TITLE
When will the matmul_4bit operator support computing gradients for B？

### ISSUE THREAD

**Issue by RanchiZhao** [TimDettmers/bitsandbytes#642]
As mentioned above, your latest code is as follows (in autograd/_functions.py, class MatMul4Bit(torch.autograd.Function):):
`    @staticmethod
    def backward(ctx, grad_output):
        if ctx.is_empty:
            bias_grad = None if ctx.bias is None else torch.zeros_like(ctx.bias)
            return torch.zeros_like(ctx.A), torch.zeros_like(ctx.B), None, bias_grad, None

        req_gradA, _, _, req_gradBias, _= ctx.needs_input_grad
        A, B = ctx.tensors
        state = ctx.state

        grad_A, grad_B, grad_bias = None, None, None

        if req_gradBias:
            # compute grad_bias first before changing grad_output dtype
            grad_bias = grad_output.sum(0, dtype=ctx.dtype_bias)

        # not supported by PyTorch. TODO: create work-around
        #if req_gradB: grad_B = torch.matmul(grad_output.t(), A)
        if req_gradA: grad_A = torch.matmul(grad_output, F.dequantize_fp4(B, ctx.state).to(grad_output.dtype).t())
        
        return grad_A, grad_B, None, grad_bias, None`
